### ISSUE TITLE
apollo-server-core@3.6.0 is trying to import now removed apollo-graphql dependency

### ISSUE THREAD

**Issue by sushantdhiman** [apollographql/apollo-server#5980]
For fixing #5941, PR #5955 removed the `apollo-graphql` package from dependencies and inlined some code from that package. But it seems not all instances of `apollo-graphql` were removed.

For example https://github.com/apollographql/apollo-server/blob/06dd1171562564f0ca180e74a6fa5530e0389076/packages/apollo-server-core/src/plugin/usageReporting/plugin.ts#L4

Above file is still trying to import `apollo-graphql`, forcing builds to fail with following error

```js
{
      detail: undefined,
      location: {
        column: 33,
        file: 'node_modules/apollo-server-core/dist/plugin/usageReporting/plugin.js',
        length: 16,
        line: 10,
        lineText: 'const apollo_graphql_1 = require("apollo-graphql");',
        namespace: '',
        suggestion: ''
      },
      notes: [
        {
          location: null,
          text: 'You can mark the path "apollo-graphql" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.'
        }
      ],
      pluginName: '',
      text: 'Could not resolve "apollo-graphql"'
    }

```

**Comment by glasser** [comment]
Yikes, good catch. Will fix tomorrow in a 3.6.1 (about to go to sleep). Feel free to send a PR to fix the import to the newly added file if you'd like!

**Comment by glasser** [comment]
Thanks, fixed in v3.6.1.
